freqtrade/rpc/telegram.py

```diff
@@ -496,6 +498,66 @@ def _forcebuy(self, update: Update, context: CallbackContext) -> None:
         except RPCException as e:
             self._send_msg(str(e))
 
+    @authorized_only
+    def _delete(self, update: Update, context: CallbackContext) -> None:
+        """
+        Handler for /delete <id>.
+        Delete the given trade
+        :param bot: telegram bot
+        :param update: message update
+        :return: None
+        """
+
+        trade_id = context.args[0] if len(context.args) > 0 else None
+        try:
+            msg = self._rpc_delete(trade_id)
+            self._send_msg('Delete Result: `{result}`'.format(**msg))
+
+        except RPCException as e:
+            self._send_msg(str(e))
+
+    @authorized_only
+    def _trades(self, update: Update, context: CallbackContext) -> None:
+        """
+        Handler for /trades <n>
+        Returns last n recent trades.
+        :param bot: telegram bot
+        :param update: message update
+        :return: None
+        """
+        stake_cur = self._config['stake_currency']
+        fiat_disp_cur = self._config.get('fiat_display_currency', '')
```

Comment:
Once tests pass - these 2 lines should be removed. They're not used and flake8 will freak about them :laughing: